src/swagger/accommodation.swagger.js

```diff
@@ -87,3 +87,56 @@
  *       '401':
  *         description: No valid token supplied
  */
+
+
+/**
+* @swagger
+* definitions:
+*   AccommodationLike:
+*     type: object
+*     properties:
+*       like:
+*         type: string
+*     required:
+*         - like
+*/
+/**
+ * @swagger
+ * /api/accommodations/{accommodationId}/like-unlike:
```

Comment:
@higustave-ops we need either two endpoints /like and /dislike or one endpoint and add put like/dislike values in the body.